e2e-tests/functions

```diff
@@ -483,12 +483,20 @@ get_service_endpoint() {
 get_metric_values() {
     local metric=$1
     local instance=$2
+    local user_pass=$3
     local start=$($date -u "+%s" -d "-1 minute")
     local end=$($date -u "+%s")
     local endpoint=$(get_service_endpoint monitoring-service)
+    if [[ "x${user_pass}" != "x" ]]; then
+        curl -s -k "https://${user_pass}@$endpoint/graph/api/datasources/proxy/1/api/v1/query_range?query=$metric%7Bcontainer_name%3D%22$instance%22%7D&start=$start&end=$end&step=60" \
+            | jq '.data.result[0].values[][1]' \
+            | grep '^"[0-9]'
+        return 0
+    fi
     curl -s -k "https://$endpoint/graph/api/datasources/proxy/1/api/v1/query_range?query=$metric%7Binstance%3D%22$instance%22%7D&start=$start&end=$end&step=60" \
```

Comment:
will it work if we pass $user_pass in regular curl command without "if"?
"@" symbol shouldn't be an issue.
```suggestion
     curl -s -k "https://${user_pass}@$endpoint/graph/api/datasources/proxy/1/api/v1/query_range?query=$metric%7Binstance%3D%22$instance%22%7D&start=$start&end=$end&step=60" \
```